Python/challenges/day_2.py

```diff
@@ -0,0 +1,23 @@
+def get_result(round, strategy=False):
```

Comment:
What if there's a third strategy?

Reply:
```suggestion
def get_result(round, strategy=False, strategy2=False, ...):
```

Reply:
How will you handle someone calling `get_result("A X", True, True)`

Reply:
they will simply only have the output of the first strat as it returns you out of the function